app/javascript/src/components/login/SignUpFields.jsx

```diff
@@ -77,7 +78,9 @@ class SignUpFields extends Component {
     const steps = [
         { key: 'user', icon: 'user', title: 'User Credentials', description: 'Add your email and create an account password.', active: (activeStep === 0) },
         { key: 'team', active: true, icon: 'users', title: 'Team Information', description: 'Let us know which team you are on!', active: (activeStep === 1) },
-        { key: 'join', disabled: true, icon: 'checkmark box', title: 'All Set!', active: (activeStep === 2), completed: (activeStep === 2) },
+  //      { key: 'avail', disabled: true, icon: 'clock', title: 'Availability', active: (activeStep === 2), completed: (activeStep === 2) },
+        { key: 'avail', disabled: true, icon: 'clock', title: 'Availability', active: (activeStep === 2) },
```

Comment:
Nice job adding to the steps properly. Add a description and make the key the full word `availability` rather than `avail` just to avoid any possible future confusion.